vision/product_search/product_search_create_product.rb

```diff
@@ -0,0 +1,45 @@
+# Copyright 2018 Google, LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# [START vision_product_search_create_product]
+# [START vision_product_search_tutorial_import]
+require "google/cloud/vision"
+# [END vision_product_search_tutorial_import]
+
+
+def product_search_create_product project_id = "your-project-id"
+  client = Google::Cloud::Vision::ProductSearch.new
+
+  # A resource that represents Google Cloud Platform location.
+  location = "us-west1"  # specify a compute region name
+  location_path = client.class.location_path project_id, location
+
+  # Create a product with the product specification in the region.
+  # Set product display name and product category.
+  product = {
+    display_name:     "sample-product-1234",
+    description:      "Athletic shorts",
+    product_category: "apparel",
+    product_labels:   [{key: "color", value: "blue"}]
```

Comment:
```ruby
[{ key: "color", value: "blue" }]
```

Add padding inside `{` and `}` so characters aren't touching the hash sides

Another Pythonism we don't use :)

https://github.com/github/rubocop-github/blob/master/STYLEGUIDE.md#collections